src/common/icons.js

```diff
@@ -29,8 +29,17 @@ const rankIcon = (rankIcon, rankLevel, progress) => {
       `;
     case "progress":
       return `
-      <text x="-5" y="3" alignment-baseline="central" dominant-baseline="central" text-anchor="middle" data-testid="progress-rank-icon" class="rank-progress-text">
-          ${progress.toFixed(1)}%
+        <text x="-5" y="3" alignment-baseline="central" dominant-baseline="central" text-anchor="middle" data-testid="progress-rank-icon" class="rank-progress-text">
+            ${(100 - percentile).toFixed(1)}%
+        </text>
+        `;
+    case "percentile":
+      return `
+      <text x="-5" y="-12" alignment-baseline="central" dominant-baseline="central" text-anchor="middle" data-testid="percentile-top-header" class="rank-percentile-header">
```

Comment:
The `y` values were chosen after a quick look. It could be that these could be improved to align the header and value better.